modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -61,6 +63,12 @@ public class UpgradeCatchAllCheck {
 		_htmlParser.extractText(null);
 
 		new ArrayList<>(LanguageUtil.getAvailableLocales());
+
+		CookiesConstants.NAME_COMMERCE_CONTINUE_AS_GUEST;
+
+		CookiesConstants.NAME_ID;
+
+		CookiesConstants.NAME_USER_UUID;
```

Comment:
@MicaelleSilvaa, do we really need these blank lines between the constants? If not, please remove them :wink:

Reply:
Resolved :)